### PR TITLE
Allow restricted admins and users to view global settings.

### LINKED ISSUE #4954: CONFIGURATION -> Global Settings menu item missing for Restricted Admins users
Internal reference: SURE-3906

*Issue description:*

Restricted Admin users are unable to see the Global Settings menu item under CONFIGURATION

*Business impact:*

Customers using Hosted Rancher cannot view global settings in the UI

*Troubleshooting steps:*
none

*Repro steps:*
 * Create a restricted admin user.
 * Log in as the restricted admin user.
 * Click the menu icon in the top left corner

*Workaround:*
 Is workaround available and implemented? 
- yes
 What is the workaround: 
- You can navigate directly to the URL, for example https://support.rancher.cloud/dashboard/c/c-p24tr/settings/management.cattle.io.setting

*Actual behavior:*

Global Settings menu is missing from UI for restricted admin users.

*Expected behavior:*

Global Settings menu should be available for restricted admin users. See also [https://rancher.com/docs/rancher/v2.6/en/admin-settings/rbac/global-permissions/] (Under Settings, Restricted Admin should be able to list, but not edit, settings)

### PR DESCRIPTION
**Pull request by Shavindra**
### Summary

Fixes #4954 - Show global settings for users 

### Technical notes summary
 - Remove ` ifHaveVerb:          'PUT'1  from `config/product/settings.js`.
 - Disable buttons
 
### Areas or cases that should be tested
 - Global settings for different users e.g. `admin`, `restricted-admin`, `user`

### CODE REVIEW

File: store/type-map.js [hunk 1]
```diff
@@ -1251,6 +1251,22 @@ export const getters = {
           return false;
         }
 
+        if (p.overrideIfHaveVerbFor) {
```
Comment (richard-cox) [inline store/type-map.js:1254]: When this is generic it can be bumped up out of the `ifHaveType` condition

File: store/index.js [hunk 2]
```diff
@@ -537,6 +537,10 @@ export const actions = {
       promises['settings'] = dispatch('management/findAll', { type: MANAGEMENT.SETTING });
     }
 
+    if ( getters['management/canList'](MANAGEMENT.GLOBAL_ROLE_BINDING) ) {
```
Comment (richard-cox) [inline store/index.js:540]: Ok, I see the gotcha here, I failed to realise we'd need to fetch all global role bindings up front.

Need to decide which is preferable to resolve https://github.com/rancher/dashboard/issues/4954

1. Load global role bindings on every visit 
- could impact systems with lots of people/project 
- could be a nice feature for other use cases
2. Show Global Settings to everyone
- reveal admin only things to non-admins 
- they can do this already, but not in the UI

**Review by richard-cox — CHANGES_REQUESTED** [review]
Ok, this approach won't work, sorry for suggesting it. See https://github.com/rancher/dashboard/issues/4954#issuecomment-1108573899

**Comment by richard-cox** [comment]
As per https://github.com/rancher/dashboard/pull/5697#pullrequestreview-951880854, this approach unfortunately won't work

**Review by richard-cox — CHANGES_REQUESTED** [review]
As per https://github.com/rancher/dashboard/pull/5697#pullrequestreview-951880854, this approach unfortunately won't work.

**Comment by Shavindra** [comment]
@richard-cox yes... I am disabling `rowsActions`. I am leveraging existing functionality

https://github.com/rancher/dashboard/blob/master/components/SortableTable/index.vue#L931-L937

**Review by richard-cox — CHANGES_REQUESTED** [review]
Banners & Branding pages `Apply` buttons are still shown when the user doesn't have permissions to edit

**Review by richard-cox — APPROVED** [review]

**Pull request merged**